src/frontend/src/lib/constants/test-ids.constants.ts

```diff
@@ -72,3 +75,6 @@ export const VIP_QR_CODE_COPY_BUTTON = 'vip-qr-code-copy-button';
 export const VIP_CODE_REGENERATE_BUTTON = 'vip-code-regenerate-button';
 
 export const SOL_TRANSACTION_SKELETON_PREFIX = 'sol-txn';
+
+export const MANAGE_TOKEN_LIST_SAVE = 'manage-token-list-save';
+export const NETWORK = 'network';
```

Comment:
Maybe too generic? A better name could be 

```suggestion
export const NETWORKS_SWITCHER_SELECTOR = 'networks-switcher-selector';
```